src/parcsr_ls/par_mgr_solve.c

```diff
@@ -787,7 +797,7 @@ hypre_MGRCycle( void               *mgr_vdata,
                hypre_ParCSRMatrixMatvecOutOfPlace(-1.0, A_array[level],
                                                   U_array[level], 1.0, F_array[level], Vtemp);
 
-               resnorm = hypre_ParVectorInnerProd(Vtemp, Vtemp);
+               resnorm = sqrt(hypre_ParVectorInnerProd(Vtemp, Vtemp));
```

Comment:
@oseikuffuor1 This seems to be a bug before the fix. Do you agree? Thanks @victorapm

Reply:
Yes, subtle but important fix. Thanks @victorapm.